packages/synapse-react-client/src/utils/challenge/evaluation/EvaluationUtils.ts

```diff
@@ -0,0 +1,47 @@
+import { EvaluationRound } from '@sage-bionetworks/synapse-client'
+import dayjs from 'dayjs'
+
+/**
+ * Given a list of evaluation rounds:
+ *  - If there is a current round, return it
+ *  - Else if there is one or more upcoming rounds, return the soonest
+ *  - Else return the most recent round in the past
+ * @param rounds
+ */
+export function getCurrentOrNextOrLatestRound(
```

Comment:
We have to decide which round to pick when we show the start/end/limits. I think this algorithm aligns with what users might expect.